app/helpers/course/controller_helper.rb

```diff
@@ -27,4 +27,23 @@ def link_to_course_user(user, options = {})
       end
     end
   end
+
+  # Links to the given User or CourseUser. If a User is given, the CourseUser under
+  # current_course of the given user will be displayed.
+  #
+  # @param [CourseUser|User] user The CourseUser/User to display.
+  # @param [Hash] options The options to pass to +link_to+
+  # @param [Proc] block The block to use for displaying the user.
+  # @return [String] The user-visible string, including embedded HTML which will display the
+  #   string within a link to bring to the User page.
+  def link_to_user(user, options = {}, &block)
+    course = controller.try(:current_course)
```

Comment:
actually no need to try, this helper is only used with Course::Controller, which has current_course.